definitions/scenarios/upgrade_to_satellite_6_12.rb

```diff
@@ -73,6 +73,7 @@ class PostMigrations < Abstract
     def compose
       add_step(Procedures::RefreshFeatures)
       add_step(Procedures::Service::Start.new)
+      add_step(Procedures::Pulpcore::TrimRpmChangelogs.new)
```

Comment:
Will this also do it for z-stream of 6.12? If not, I think we should.

Reply:
No it will not, can add!

Reply:
Done